### ISSUE TITLE
Render fails on Windows when trying to pull some remote asset with a dot in the URL

### ISSUE THREAD

**Issue by pierreminiggio** [JonnyBurger/remotion#299]
Hello !
So I'm pulling assets (audio & video) from remote urls.
While rendering on Windows, one of these audio assets (always the same one), keeps failing to get downloaded :
This asset is this one : https://gtts-api.miniggiodev.fr/Ici+Japon+Corp.?lang=ja

This issue doesn't occur during preview, only while rendering.

Here is the error : 
![image](https://user-images.githubusercontent.com/35456349/115617459-b4dafa80-a2f1-11eb-83f7-71fa93f7032c.png)
```console
Error: Command failed with exit code 1: ffprobe -v error -show_entries stream=channels -of default=nw=1 C:\Users\pierr\AppData\Local\Temp\react-motion-graphicsJVUDZl\Ici+Japon+Corp.
C:\Users\pierr\AppData\Local\Temp\react-motion-graphicsJVUDZl\Ici+Japon+Corp.: No such file or directory
    at makeError (F:\dev\youtube-likes-recap-video-maker\node_modules\execa\lib\error.js:59:11)
    at handlePromise (F:\dev\youtube-likes-recap-video-maker\node_modules\execa\index.js:114:26)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Object.getAudioChannels (F:\dev\youtube-likes-recap-video-maker\node_modules\@remotion\renderer\dist\assets\get-audio-channels.js:17:18) {
  shortMessage: 'Command failed with exit code 1: ffprobe -v error -show_entries stream=channels -of default=nw=1 C:\\Users\\pierr\\AppData\\Local\\Temp\\react-motion-graphicsJVUDZl\\Ici+Japon+Corp.',
  command: 'ffprobe -v error -show_entries stream=channels -of default=nw=1 C:\\Users\\pierr\\AppData\\Local\\Temp\\react-motion-graphicsJVUDZl\\Ici+Japon+Corp.',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: '',
  stderr: 'C:\\Users\\pierr\\AppData\\Local\\Temp\\react-motion-graphicsJVUDZl\\Ici+Japon+Corp.: No such file or directory',
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
```

The only thing that I think differenciate this asset from the others is the random dot in the URL.

So I went ahead and tried to render again without the dot, and it worked !

I could filter out the dot myself while creating my link to fix the issue because it's not needed for my usecase, but maybe there is something to be done if someone wanna pull an asset that actually has a dot in its link.

I'm currently running the render as well in a github action & on a Linux server to see if the issue occurs there as well or if that's only on Windows.
EDIT : The Github action doesn't seem to produce an error. And when trying to rendering on a Linux Server I don't get the error as well. So it's likely a Windows only bug.

To reproduce this bug, you can clone this repo and start a build : https://github.com/pierreminiggio/youtube-likes-recap-video-maker/tree/c612624a1de7ae597ced49c4fc10de32577d08cd

**Comment by JonnyBurger** [comment]
Thanks for the excellent bug report! Fix will be merged and released by today evening, look out for v2.0.4!

**Comment by pierreminiggio** [comment]
> Thanks for the excellent bug report! Fix will be merged and released by today evening, look out for v2.0.4!

Okey thanks ! I'll try in v2.0.4 and report here if that fixes it.
